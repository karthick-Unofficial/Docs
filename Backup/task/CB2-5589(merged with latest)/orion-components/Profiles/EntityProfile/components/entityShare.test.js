"use strict";

var _interopRequireDefault = require("@babel/runtime-corejs3/helpers/interopRequireDefault");
var _react = _interopRequireDefault(require("react"));
var _EntityShare = _interopRequireDefault(require("./EntityShare"));
var _enzyme = require("enzyme");
describe("EntityShare", function () {
  it("renders", function () {
    var wrapper = (0, _enzyme.shallow)( /*#__PURE__*/_react["default"].createElement(_EntityShare["default"], null));
    expect(wrapper).toMatchSnapshot();
  });
});