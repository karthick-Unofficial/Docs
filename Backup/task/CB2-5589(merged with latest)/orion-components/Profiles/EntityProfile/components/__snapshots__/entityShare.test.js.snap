// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EntityShare renders 1`] = `
<ForwardRef(Dialog)
  actions={
    Array [
      <ForwardRef(Button)
        color="primary"
        variant="text"
      >
        share
      </ForwardRef(Button)>,
      <ForwardRef(Button)
        color="primary"
        variant="text"
      >
        cancel
      </ForwardRef(Button)>,
    ]
  }
  className="share-dialog"
>
  <p>
    <Translator
      value="global.profiles.entityProfile.entityShare.confirmationText"
    />
  </p>
</ForwardRef(Dialog)>
`;
